spec/requests/questions_spec.rb

```diff
@@ -81,6 +101,34 @@
     end
   end
 
+  describe "PATCH update with invalid parameters" do
+    let(:question) do
+      quiz.questions.create!(
+        content: "One",
+        option_a: "Answer A",
+        option_b: "Answer B",
+        option_c: "Answer C",
+        option_d: "Answer D",
+        correct_option: "option_a",
+      )
+    end
+
+    it "does not update the question and re-renders the edit form" do
+      patch quiz_question_path(quiz, question), params: { question: {
+        content: "",
+        option_a: "",
+        option_b: "Answer B",
+        option_c: "Answer C",
+        option_d: "Answer D",
+        correct_option: "option_a",
```

Comment:
```suggestion
```

Since this is a patch request you shouldn't need to pass in all the attributes (Patch requests are able to update just a subset of attributes)